package.json

```diff
@@ -7,9 +7,12 @@
     "bin": "bin"
   },
   "scripts": {
+    "start": "node ./bin/server.js",
     "test": "mocha",
-    "lint": "eslint bin/** lib/** test/**",
-    "install": "cd pkcs7 && bundle install"
+    "cover": "rm -Rf coverage/* && istanbul cover _mocha -- -R spec",
+    "lint": "eslint bin/* lib/*.js lib/**/*.js test/*.js test/**/*.js",
+    "report": "coveralls < ./coverage/lcov.info",
+    "warden-mock": "node ./test/bin/warden.js"
```

Comment:
copy pasta but forgot to save changes made to the script. 
updated script just pushed.